styles.css

```diff
@@ -0,0 +1,233 @@
+:root {
+  --primary: #172b4d;
+  --h-color: #3a4a42;
+  --secondary: #36b37e;
+  --secondary-btn: #36b37f;
+  --weight: bold;
+  --background: #ebf0ee;
+}
+
+* {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+  overflow-x: hidden;
+}
+
+a {
+  text-decoration: none;
+  color: white;
+}
+
+.logo {
+  text-decoration: none;
+  font-size: 1.6rem;
+  color: #28352f;
+}
+
+body {
+  width: 100%;
+  height: 100%;
+  font-family: "Inter", sans-serif;
+  overflow-x: hidden;
+  line-height: 1.5;
+  -webkit-font-smoothing: antialiased;
+}
+
+header {
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+  background-color: #fff;
+}
+
+header h2 {
+  animation: crane-float 2s alternate 5 ease-in-out both;
+}
+
+nav .nav-menu {
+  visibility: visible;
+  transform: translateY(0%);
+  display: flex;
+  justify-self: center;
+}
+
+nav .nav-menu .nav-link {
+  font-size: 32px;
+  line-height: 44px;
+  font-weight: 600;
+  margin: 12px;
+}
+
+nav .nav-menu .nav-link a {
+  display: flex;
+  justify-self: center;
+  padding: 5px 15px;
+  margin: 3px;
+  font-size: 16px;
+  color: var(--primary);
+  font-weight: 500;
+  text-decoration: none;
+  cursor: pointer;
+}
+
+.main-section {
+  background-image: url("./media/header-shapes\ mobile@2x.svg");
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-position: center top;
+  width: 100%;
+  margin: 0 auto;
+  text-align: center;
+  padding: 0 24px;
+}
+
+.main-section-title {
+  padding-top: 54px;
+  animation: top-bottom 1s ease-in-out;
```

Comment:
- [ ] These animations are not working properly, it is also not a requirement to use animations in this project. If you want to learn how to properly use them go [here](https://www.w3schools.com/css/css3_animations.asp). 😄